include/ausy_encoder.h

```diff
@@ -0,0 +1,11 @@
+#ifndef AUSY_ENCODER_H
```

Comment:
Maybe we should put a template for a public header somewhere, I usually copy it from some other file, but we have different version over the code base, so I am not sure which copyrights/trademarks are still relevant and should be included in new files. :confused: